flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/catalog/CatalogTableITCase.scala

```diff
@@ -877,6 +877,397 @@ class CatalogTableITCase(isStreamingMode: Boolean) extends AbstractTestBase {
     expectedProperty.put("k2", "b")
     assertEquals(expectedProperty, database.getProperties)
   }
+
+  @Test
+  def testCreateViewIfNotExistsTwice(): Unit = {
```

Comment:
add a test case like ` CREATE VIEW IF NOT EXISTS T3(d, e, f) AS SELECT * FROM T1`

Reply:
The second `CREATE VIEW` confirms it's ignored (silently). 
If we use `CREATE VIEW IF NOT EXISTS T3(d, e, f) AS SELECT * FROM T1`, It's hard to confirm it's ignored or it replace the first same one.
What do you think?